docs/zk2kraft.html

```diff
@@ -15,14 +15,11 @@
  limitations under the License.
 -->
 
-<script><!--#include virtual="js/templateData.js" --></script>
-
-<script id="zk2kraft-template" type="text/x-handlebars-template"></script>
-
+<!--#include virtual="../includes/_header.htm" -->
+<!--#include virtual="../includes/_top.htm" -->
+<div class="content">
 <div class="p-zk2kraft">
-    <h4 class="anchor-heading">Significant Changes in Kafka 4.0 Release</h4>
-    <p>The following are some of the updates in Kafka 4.0 release:</p>
-    <h5 class="anchor-heading">Removal Zookeeper configs</h5>
+    <h2 class="anchor-heading">Removal Zookeeper Configurations</h2>
```

Comment:
@frankvicky Just a tiny suggestion to improve clarity:

```suggestion
    <h2 class="anchor-heading">Removing Zookeeper Configurations</h2>
```
or Zookeeper Configurations `Removal` or `Removal` of Zookeeper Configurations